.evg.yml

```
@@ -111,9 +113,122 @@ tasks:
     commands:
       - func: "generate sbom"
       - func: "upload sbom"
+      - func: "push SBOM Lite to Silk"
+      - func: "pull augmented SBOM from Silk"
       - func: "scan sbom"
 
+  - name: ssdlc-artifacts
+    run_on: ubuntu2204-small
+    depends_on:
+      - name: sbom
+        variant: code-quality-and-correctness
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "pull augmented SBOM from Silk"
+
+  - name: publish-augmented-sbom
+    run_on: ubuntu2204-small
+    git_tag_only: true
+    depends_on:
+      - name: "publish-maven"
+        variant: "release"
+      - name: ssdlc-artifacts
+        variant: "release"
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "publish augmented SBOM"
```

```suggestion
  - name: ssdlc-artifacts
    run_on: ubuntu2204-small
    depends_on:
      - name: sbom
        variant: code-quality-and-correctness
    exec_timeout_secs: 300 # 5m
    commands:
      - func: "pull augmented SBOM from Silk"

  - name: publish-augmented-sbom
    run_on: ubuntu2204-small
    git_tag_only: true
    depends_on:
      - name: "publish-maven"
        variant: "release"
      - name: ssdlc-artifacts
        variant: "release"
    exec_timeout_secs: 300 # 5m
    commands:
      - func: "publish augmented SBOM"
```
  - name: ssdlc-artifacts-snapshot
    run_on: ubuntu2204-small
    depends_on:
      - name: sbom
        variant: code-quality-and-correctness
    exec_timeout_secs: 300 # 5m
    commands:
      - func: "publish augmented SBOM"
```
and add `ssdlc-artifacts-release` in the `release` variant,  `ssdlc-artifacts-snapshot` in  the`code-quality-and-correctness` variant (so that it won't run at release time) or put a tag on `ssdlc-artifacts-snapshot` and we can exclude the task with that tag via Evergreen